lib/dfe/reference_data/degrees/types.rb

```diff
@@ -40,9 +41,11 @@ module Degrees
             suggestion_synonyms: [],
             match_synonyms: [
               'BA Hons',
-              'BA (Hons)', 'BA Honours',
+              'BA (Hons)',
+              'BA Honours',
               'Bachelor of Arts with honours',
-              'Bachelor of Arts (Hons)'
+              'Bachelor of Arts (Hons)',
+              'Bachelo of Arts (Honours)'
```

Comment:
Isn't this one just a typo?

Reply:
Typo actually on my side ;) – now fixed.